.orchestra/ci/binary-archives-hook.sh

```diff
@@ -7,6 +7,11 @@ set -euo pipefail
 #
 # Mandatory parameters:
 #
+# 1:
```

Comment:
```
./binary-archives-hook.sh CHANGED_FILES

CHANGED_FILES
   File containing the list of changed files in the binary archives, each line
   is a different file path

Environment variables:

  COMPONENT_TARGET_BRANCH: target branch the CI has been triggered against

[...]
```

Same elsewhere, the environment variables are list under "parameters", which is confusing.

Also, please document in the header what this script *does*. It only explains when it runs.